sdks/python/apache_beam/coders/coders.py

```diff
@@ -1485,6 +1486,21 @@ def as_cloud_object(self, coders_context=None):
         ],
     }
 
+  def to_type_hint(self):
+    from apache_beam.typehints import sharded_key_type
```

Comment:
Why isn't this import at the top of the file?

Reply:
To avoid circular import.